tests/web/conftest.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2016 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this licence, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+"""Pytest configuration for web tests."""
+
+from __future__ import absolute_import, division, print_function
+
+from os import environ
+from time import sleep
+
+import pytest
+
+from selenium import webdriver
+
+@pytest.yield_fixture(scope='session')
+def driver():
+    """Selenium driver."""
+
+    desired_cap = desired_cap = {
+        'platform': "Mac OS X 10.11",
+        'browserName': "firefox",
+        'version': "48",
+        'tunnel-identifier': environ['TRAVIS_JOB_NUMBER']
+    }
```

Comment:
In principle yes, although, I am not sure we want to have too much of it, as it would slow down Travis even further. Maybe we can have just have one mobile and one firefox.